website/content/docs/overview/what-is-boundary.mdx

```diff
@@ -43,77 +38,20 @@ The core Boundary workflow consists of four stages:
-
-When [session recording](/boundary/docs/configuration/session-recording) is enabled on a target, any sessions established to that target are recorded.
-Recorded sessions are stored in an admin-specified storage backend.
-Administrators can access recorded sessions at a later time using a session player that runs in a browser.
-
-### Filtering
-
-Parts of Boundary support filters for various purposes. For a description
-of the filter syntax, see the [filtering](/boundary/docs/concepts/filtering) page. See
-the docs pages for the individual resources or capabilities where filters are
-supported for the specific inputs and examples with those inputs.
+- **Access:** Boundary streamlines connection to hosts by automating discovery and access configuration as workloads are deployed or changed.
 
--> **Tip**: Learn more about Boundary [use cases](/boundary/docs/overview/use-cases).
+## Which edition of Boundary is right for me?
 
-## What is HCP Boundary?
+- **HCP Boundary**: a managed offering of Boundary with commercial-only use-cases features. Boundary's control plane is hosted by HashiCorp and customers have the option of running private workers within their environment.
+- **Boundary OSS**: an open source, self-managed version of Boundary.
+- **Boundary Enterprise**: a self-managed offering of Boundary with full feature parity to HCP Boundary.
 
-Boundary offers three types of deployment options.
-The first two options are self-managed deployments.
-A self-managed approach enables organizations to proxy all session data through their own networks, while still providing the convenience of a managed service.
-HashiCorp offers Boundary in both open source (OSS) and enterprise self-managed versions.
-The Enterprise version contains commercial-only features, and comes with support.
+If you're not sure which edition is right for you, we recommend [**HCP Boundary**](https://developer.hashicorp.com/boundary/tutorials/hcp-getting-started) because it eliminates deployment operations.
 
-The third option is an *HCP-managed* deployment solution where both the control plane and worker nodes are managed by HashiCorp. With this managed solution, an option of private workers is offered. **HCP Boundary** is a fully-managed, cloud-based workflow that enables secure connections to remote hosts and critical systems across cloud and on-premise environments. Refer to the [HCP Boundary](/hcp/docs/boundary) documentation to learn more.
+Regardless of which server edition of Boundary you are hosting, all editions use the same Desktop and CLI clients, which you can download [here](https://developer.hashicorp.com/boundary/downloads).
 
--> **Hands On:** Try the [Create a Boundary Instance on HCP](/boundary/tutorials/hcp-getting-started/hcp-getting-started-create) tutorial to deploy an HCP Boundary instance.
 
-## Tutorial
+## Getting Started
```

Comment:
This Getting Started section feels redundant to the new "Get Started here" link that was added to the introduction. I think we should delete one of them.

Reply:
Will do